app/views/standard/confirmDeleteAdditionalMetadata.scala.html

```diff
@@ -1,27 +1,32 @@
 @import views.html.partials.{fileNameCard, warningMessage}
 
 @import java.util.UUID
+@import views.html.helper.CSRF
+@import views.html.helper.form
 @(consignmentId: UUID, metadataType: String, fileIds: List[UUID], fileNames: List[String], name: String)(implicit messages: Messages, request: RequestHeader)
     @main(title = s"Delete $metadataType metadata", name = name) {
         @defining(play.core.PlayVersion.current) { version =>
 
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-two-thirds">
-                    <h1 class="govuk-heading-xl">
-                        Delete @metadataType metadata
-                    </h1>
-                    @fileNameCard(fileNames, Messages(s"deleteAdditionalMetadata.${metadataType}DeletionMessage"))
-                    @warningMessage(s"Once deleted $metadataType metadata cannot be recovered.")
-                    <p class="govuk-body">Are you sure you would like to proceed?</p>
+                    @form(routes.DeleteAdditionalMetadataController.deleteAdditionalMetadata(consignmentId, metadataType, fileIds), (Symbol("novalidate"), "")) {
```

Comment:
Is there a reason for changing this to a form?

Reply:
The "delete and return to files" is now a button rather than a link so I had to define a route for the submit.